### ISSUE TITLE
Optional chaining introduced in TS 3.7 not supported?

### ISSUE THREAD

**Issue by schlundus** [SonarSource/SonarTS#894]
Sonar raises an issue for optional chaining stuff introduced in TS 3.7

**SonarTS version:** 2.1 (build 4359)<br>
**TypeScript version:** 3.7.3<br>

**SonarQube version:** 7.9.1.27448
--

**Rule key:** typescript:S905

**Reproducer** 
```
// minimal reproducer if relevant
something like this
foo.bar?.addTo(map);
```

**Expected behavior**
No issue raised

**Comment by saberduck** [comment]
hello,

typescript analysis is now done with [SonarJS plugin](https://github.com/SonarSource/sonarjs), so the proper place to report it is there. However, I didn't manage to reproduce your issue. Which version of SonarJS is installed?
